connector/src/test/scala/com/google/cloud/spark/bigquery/it/SparkBigQueryEndToEndReadFromQueryITSuite.scala

```diff
@@ -67,6 +67,28 @@ class SparkBigQueryEndToEndReadFromQueryITSuite extends FunSuite
     corpuses should equal(expectedCorpuses)
   }
 
+  def testReadFromQueryWithNewLine(format: String) {
```

Comment:
Can we reuse the code from method **testReadFromQuery** (at line 51) as only the sql statement is different between these two methods and rest of the code is same?

Reply:
Also, please add space between **created** and **due** in the comment